tests/evm_backends/abi_contract.py

```diff
@@ -136,6 +140,7 @@ def argument_types(self) -> list:
     def argument_count(self) -> int:
         return len(self.argument_types)
 
+    # REVIEW: why not `_signature`?
```

Comment:
maybe args_signature or args_abi_type then

Reply:
I renamed to `_args_signature` and this can be prepare_calldata after fixing the method_id (https://github.com/vyperlang/vyper/pull/3846/commits/3ab187f00599db78d79b5a738974f5f43f6e079c)